projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/TestJobDeployTemporaryCredsIT.java

```diff
@@ -133,33 +137,17 @@ public void testDeployment() throws Exception {
                 .contentType(MediaType.APPLICATION_JSON))
         .andExpect(status().isAccepted());
 
+    Thread.sleep(60000); // Since uploading image to ecr is asynchronous we have to wait.
```

Comment:
better use the following technique:

    // Wait for the job execution to complete, polling every 5 seconds
    // See: https://github.com/awaitility/awaitility/wiki/Usage
    await()
        .atMost(10, TimeUnit.MINUTES)
        .with()
        .pollInterval(15, TimeUnit.SECONDS)
        .until(terminationMetricsAreAvailable());